test/Serilog.Tests/Support/DelegatingSink.cs

```diff
@@ -20,7 +20,7 @@ public void Emit(LogEvent logEvent)
 
         public static LogEvent GetLogEvent(Action<ILogger> writeAction)
         {
-            LogEvent result = null;
+            LogEvent result = null!;
```

Comment:
I think I missed this one

Reply:
lol. why are they all named different ;)